website/vercel.json

```diff
@@ -2,6 +2,11 @@
   "cleanUrls": true,
   "trailingSlash": false,
   "redirects": [
+    {
+      "source": "/reference/global-configs/legacy-behaviors#adapter-specific-behavior-changes",
+      "destination": "reference/global-configs/adapter-behavior-changes",
+      "permanent": true
```

Comment:
might want to test this redirect in the pr preview. the `destination` value doesn't look right to me :scratches-head:

Reply:
THat was the original page title. Good catch